graylog2-web-interface/src/logic/rest/FetchProvider.js

```diff
@@ -54,7 +74,7 @@ export class Builder {
     return this;
   }
 
-  json(body) {
+  json(body, handleForbidden = redirectIfForbidden, handleUnauthorized = logoutIfUnauthorized) {
```

Comment:
Instead of having to pass `handleForbidden`/`handleUnauthorized` to every method, we should introduce two builder methods for these, which allows consumers to set these. Creating a new `FetchBuilder` would default to `redirectIfForbidden` respectively `logoutIfUnauthorized`.